jobs/JEVS_CAM_PLOTS

```diff
@@ -79,15 +81,16 @@ fi
 export COMROOT=${COMROOT:-$(compath.py $envir/com)}
 
 if  [ $MODELNAME = href ] ; then
-  export COMIN=${COMIN:-$(compath.py  $envir/com/$NET/$evs_ver)}/stats/$COMPONENT/$MODELNAME.$VDATE
+  export COMIN=${COMIN:-$(compath.py $envir/com/$NET/$evs_ver)}/stats/$COMPONENT/$MODELNAME.$VDATE
   export COMOUT=${COMOUT:-$(compath.py -o $envir/com/$NET/${evs_ver})}/$STEP/$COMPONENT/$RUN.$VDATE
-  export COMINapcp24mean=${COMINapcp24mean:-$(compath.py -o $envir/com/$NET/$evs_ver)}/stats/$COMPONENT
+  export EVSINapcp24mean=${EVSINapcp24mean:-$COMIN}/stats/$COMPONENT
```

Comment:
I have begun running plots and `jevs_cam_href_precip_spatial_plots.sh` produced an empty tar b/c EVSINapcp24mean was not resolved correctly. This path needs to end at $COMPONENT, but $COMIN gets redefined in line 84 to include $MODELNAME.$VDATE. I tested the following and it runs correctly.  Please modify line 86. dirname will strip $MODELNAME.$VDATE.
```
export EVSINapcp24mean=${EVSINapcp24mean:-$(dirname $COMIN)}
```

Reply:
Oops, okay, made the change.  Thanks for catching this.